vaccine_feed_ingest/runners/al/arcgis/normalize.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+# isort: skip_file
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from datetime import datetime
+from typing import Optional, List
+
+from vaccine_feed_ingest_schema import location
+
+from vaccine_feed_ingest.utils.validation import BOUNDING_BOX
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("al/arcgis/normalize.py")
```

Comment:
Please use our [shared logger config (see example)](https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/d6e54ca6c32a32ab4c04d47146f1a7bab33c26cf/vaccine_feed_ingest/runners/az/pinal_ph_vaccinelocations_gov/normalize.py#L12-L14) instead.

Reply:
Nifty! The logger has been updated. I've also changed the vaccine_feed_ingest_schema import to be the same as in your linked example.